diff --git a/doc/source/conf.py b/doc/source/conf.py
index baac650..3d2c6a1 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -14,7 +14,7 @@ sys.path.insert(0, os.path.abspath('./'))
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.intersphinx',
-    'oslosphinx',
+    'oslo.sphinx',
     'ext.extract_docs'
 ]
 
diff --git a/test-requirements.txt b/test-requirements.txt
index 3488df6..50e8c3d 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -1,2 +1,2 @@
-oslosphinx
+oslo.sphinx
 sphinx>=1.1.2,<1.2
