From 2e78d652efd676c18716cfe286904f83f00765fe Mon Sep 17 00:00:00 2001
From: Angus Thomas <athomas@redhat.com>
Date: Mon, 7 Apr 2014 17:07:36 +0100
Subject: [PATCH] Default DB location

---
 etc/ironic/ironic.conf.sample | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/ironic/ironic.conf.sample b/etc/ironic/ironic.conf.sample
index 76f6bae..3f2c3b5 100644
--- a/etc/ironic/ironic.conf.sample
+++ b/etc/ironic/ironic.conf.sample
@@ -528,7 +528,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection=<None>
+connection=sqlite:////var/lib/ironic/$sqlite_db
 
 # The SQL mode to be used for MySQL sessions (default is
 # empty, meaning do not override any server-side SQL mode
-- 
1.9.0

